### PR TITLE
Preview Revamp Update to Systemd

### PR DESCRIPTION
**Pull request by cthulhuplus**
Using "service" to control start/stop/etc... for services is being deprecated in favor of using systemd.

### This pull request changes...

- Services are controlled using systemd
- No difference is noticeable by the end user

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,649 +1,648 @@
...
-      - persist_to_workspace:
-          root: ~/project
-          paths:
-            - ./build-url.txt
-      - store_artifacts:
-          path: /tmp/backend.zip
-          destination: backend.zip
-      - store_artifacts:
-          path: /tmp/frontend.zip
-          destination: frontend.zip
```
Comment (radavis) [inline .circleci/config.yml:649]: Has GitHub lost its mind? I can see a line was added (based on line numbers), but why does GitHub think the whole file has changed?

Comment (radavis) [reply .circleci/config.yml:649]: Line 596: `--AWS_AMI "$PREVIEW_AWS_AMI" \` was removed

Comment (cthulhuplus) [reply .circleci/config.yml]: Yea, I reverted a few commits because of that, some weird stuff was going on.

**Review by thetif — APPROVED** [review]

**Review by tbolt — APPROVED** [review]
Seems fine from what I can tell

**Comment by cthulhuplus** [comment]
Changes were implemented in another PR

**Pull request closed without merging**